components/mpas-ocean/driver/ocn_comp_mct.F

```diff
@@ -4092,7 +4129,16 @@ subroutine ocn_export_moab(EClock)   !{{{
             o2x_om(n, index_o2x_So_dhdy) = avgSSHGradient(index_avgMeridionalSSHGradient, i)
    
             o2x_om(n, index_o2x_Faoo_h2otemp) = avgTotalFreshWaterTemperatureFlux(i) * rho_sw * cp_sw
-   
+
+!DC, blindly following pattern
+            if (config_land_ice_flux_mode == 'standalone' .or. 'data') then
```

Comment:
@darincomeau, this should be:
```suggestion
            if (trim(config_land_ice_flux_mode)=='standalone' .or. trim(config_land_ice_flux_mode) == 'data') then
```

Reply:
ack, good catch. i think that section is for the moab coupler, not used here? But clearly a mistake, thanks I'll make the fix

Reply:
Darn!